azure/.npmignore

```diff
@@ -0,0 +1,13 @@
+node_modules/
```

Comment:
I'm guessing some of the resources (like the .PNG's) should also be in this file. Probably _not_ the SVG for the icon though.